docs/troubleshooting.md

```diff
@@ -217,3 +217,81 @@ The following is a simple example of this occurring:
+
+
+### ColletorStats counters -> rox_collector_counters
+
+| Name                                             | Description                                                                                                                          |
+|--------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| net_conn_updates                                 | Each time a connection object is updated in the model (scrapes, and kernel events).                                                  |
+| net_conn_deltas                                  | Number of connection events sent to Sensor.                                                                                          |
+| net_conn_inactive                                | Accumulated number of connections destroyed (closed)                                                                                 |
+| net_cep_updates                                  | Each time an endpoint object is updated in the model (scrapes only).                                                                 |
+| net_cep_deltas                                   | Number of endpoint events sent to Sensor.                                                                                            |
+| net_cep_inactive                                 | Accumulated number of endpoints destroyed (closed)                                                                                   |
+| net_known_ip_networks                            | Number of known-networks defined.                                                                                                    |
+| net_known_public_ips                             | Number of known public addresses defined.                                                                                            |
+| process_lineage_counts                           | Every time the lineage info of a process is created (signal emitted)                                                                 |
+| process_lineage_total                            | Total number of ancestors reported                                                                                                   |
+| process_lineage_sqr_total                        | Sum of squared number of ancestors reported                                                                                          |
+| process_lineage_string_total                     | Accumulated size of the lineage process exec file paths                                                                              |
+| process_info_hit                                 | Accessing originator process info of an endpoint with data readily available.                                                        |
```

Comment:
Strange, this and "process_info_miss" I haven't found in the actual output -- is there anything special about them?

Reply:
This one was added since last release:
https://github.com/stackrox/collector/commit/d6af1e66445ed02fadb6053fb170d603660d8e72#diff-3a441af6bffd6d75f4c735d2beb52e9a960ad83b4c478d38a5fede1824d19478